src/modules/__test__/commentsCounter.test.js

```diff
@@ -0,0 +1,8 @@
+import commentsCounter from '../commentsCounter.js';
+
+describe('CommentsCounter', () => {
+  test('Array Length should be 4', () => {
+    const arr = [1, 2, 3, 4];
+    expect(commentsCounter(arr)).toBe(4);
+  });
+});
```

Comment:
- [ ] Good Job testing all of the functions of the counters 🎉 but please note that you should test and cover all edge cases that can happen, I would suggest testing the case of having 0 items in the array where the expected result should be 0, by implementing that you will cover all of the edge cases 🙏 kindly fix that